cli/src/error.rs

```diff
@@ -29,7 +29,7 @@ pub enum CliError {
     User(String),
     Io(std::io::Error),
     Signing(String),
-    Hyper(hyper::Error),
+    Request(reqwest::Error),
```

Comment:
If this is for `reqwest` specific errors, perhaps we should name it after the crate itself (i.e., `Reqwest` instead of `Request`) ? Not sure there are other instances where `request` would come up and get confused with this error type, but it would make sense to me to make it specific to the crate rather than that word.

Reply:
I figured that since we want to keep the crates we use decoupled from our code, we should minimize the amount of code that uses crate names.